src/ansys/fluent/core/workflow.py

```diff
@@ -140,9 +145,11 @@ class BaseTask:
     """
 
     def __init__(
-        self, command_source: Union[OldWorkflowWrapper, NewWorkflowWrapper], task: str
+        self,
+        command_source: Union[ClassicWorkflowWrapper, EnhancedWorkflowWrapper],
+        task: str,
     ) -> None:
-        """Initialize BaseTask.
+        """Initialize the basetask.
```

Comment:
When accepting review suggestions, please apply critical thinking always.

Reply:
yes @seanpearsonuk. Did not notice these doc changes. Will look into it.